src/platforms/javascript/common/sourcemaps/uploading/angular-webpack.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Angular CLI
+description: "Upload your source maps using the Angular CLI, a custom Angular builder and the Sentry Webpack plugin."
+sidebar_order: 0
+supported:
+  - javascript.angular
+---
+
+You can use the Angular CLI together with our Sentry Webpack plugin to automatically create releases and upload source maps to Sentry when building your app (e.g. with `ng build`).
+Due to Angular's closed build system, you first need to to configure a custom builder to register the Webpack plugin.
+
+## Installation
+
+```shell {tabTitle:npm}
+npm install --save-dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
+
+```shell {tabTitle:yarn}
+yarn add --dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
+
+## Configuration
+
+First, make sure that you're [generating source maps](../../.#generating-source-maps) in your Angular project.
+Then, set up a [custom Angular builder](https://www.npmjs.com/package/@angular-builders/custom-webpack), to which you add the Sentry Webpack plugin.
+
```

Comment:
```suggestion
## Configure

You can configure Sentry by making sure that you're [generating source maps](../../.#generating-source-maps) in your Angular project, setting up a [custom Angular builder](https://www.npmjs.com/package/@angular-builders/custom-webpack) to which you add the Sentry Webpack plugin, and finally uploading. Configuration should happen as early as possible in your application's lifecycle.



```

Reply:
I'm not too happy with neither my first version, nor the suggestion 😅 Will try to come up with different wording. Overall, I just want to highlight what we need to configure to upload source maps (I think "configure Sentry" is a little too brought here).

Reply:
Sounds good! We can talk about this more when we meet too :)